### ISSUE TITLE
errors in wikipedia page link extraction

### ISSUE THREAD

**Issue by yuchenlin** [tatuylonen/wiktextract#22]
Hi, 

If I understand it correctly, an entry with a recommended wikipedia page like: https://en.wiktionary.org/wiki/glue_stick should have the link in its `senses` according to the readme.
But I couldn't see it in the processed results.

Thanks!

**Comment by yuchenlin** [comment]
And also I couldn't find the QID: Q1139253

**Comment by tatuylonen** [comment]
I just checked this with the latest version, and the new code also didn't properly extract the wikipedia link if it was defined under the language (as in "glue stick").  This is now fixed and I just tested that it now gets extracted correctly.  The value is the Wikipedia page name, not a URL.  The pre-extracted version at [https://kaikki.org/dictionary/](https://kaikki.org/dictionary/) should reflect this change in about 24 hours (it is updated by an overnight run).

As for Q1139253, I just tried grepping from it from all raw pages in Wiktionary, and it does not occur in Wiktionary.  QIDs should get extracted and should be under "wikidata" in when available (typically in a word sense, but they could also occur on word level).  I also tested that "United States" gets it QID (Q30) correctly extracted and under the appropriate sense.
